src/api/index.md

```diff
@@ -2165,6 +2165,16 @@ type: api
   })
   ```
 
+  Sort using two keys:
+
+  ``` html
+  <ul>
+    <li v-for="user in users | orderBy ['lastName', 'firstName']">
```

Comment:
Okay, it's funny because I did it that way at the beginning :laughing: but then I realised that in some cases you may want to apply a variable amount of parameters defined by some property
Let me know if you want me to take care of it